README.md

```diff
@@ -19,7 +19,7 @@ npm run mock
 * Sørg for at stub-oidc-provider er installert gjennom https://myapps.microsoft.com
 * Logg inn på tynnklient
 * Åpne nettleser og sørg for at du er innlogget med din egen NAV-ident
-* Gå til https://www-q0.nav.no/person/personopplysninger
+* Gå til https://www-q0.nav.no/person/behov-for-tilrettelegging/ (evt https://www.dev.nav.no/person/personopplysninger/ om tilretteleggingsbehov legges der)
```

Comment:
Vi er i usynk med parent applikasjonen i testmiljøet. Parent applikasjonen ligger i dev, med link til 
https://www-q1.nav.no/person/behov-for-tilrettelegging.
Mens vi ligger på:
https://www-q0.nav.no/person/behov-for-tilrettelegging/

Reply:
Burde vi gjøre noe her, deploye til q1 eller dev?

Reply:
Vi ønsker jo å ligge i så få testmiljø som mulig...
Men jeg tror personopplysninger-appen (parent applikasjonen) ligger i flere testmiljø.
Så det kan være at et av miljøene til den appen vil peke på en URL som vi ikke ligger på